docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,215 @@
+
+**Important:** Do not merge your own PR or any other PR. After a PR has been reviewed and approved, the code is merged by Dave, the repository maintainer. It will be merged eventually. If your PR is on a release branch and has been ready to merge for a few days, you can add a comment that it is ready to merge. 
+
+It is your responsibility to be on top of your PRs and any Issues that need follow-up.  This includes answering questions and comments, acknowledging and making change requests, and requesting re-review after you’ve addressed someone’s change requests. 
+
+This is important for the Team’s productivity -- it prevents PRs from lagging and prevents confusion as to why a PR has stalled or if you’ve seen someone’s comments.
+When issuing a PR, you can request a specific reviewer (this is not the same as assigning a PR to someone).  There is usually an obvious reviewer for a PR.  
+This is especially important in Release-Test/Fix cycles when we want PRs to be reviewed and resolved quickly.
+
+### PR Formatting
+Be sure to observe and follow team conventions for how PRs should be named and described. 
+
+**Title:** [module_name] Quick description   
+
+**[Contents](../../../.github/PULL_REQUEST_TEMPLATE.md) of the initial comment:** Include a brief summary of the changes and rationale, links to related issues or PRs, and testing instructions if there are any. 
+
+**Linking issues:** Use GitHub keywords like `Fixes` or `Resolves` so that related issues are closed automatically when the PR is merged. Do not use these keywords if you are only addressing one point raised in an issue. This will close the entire issue and leave unresolved bugs in the codebase.
+**Note** that automatic issue-linking and issue-closing only works on the default branch.  If the PR is not on the default branch, you will need to manually close the issue after the PR is merged. 
+
+**Labels:** On the right side of the browser, you will see the option to add labels. Please add any that are related to the PR. 
+**If** you are working on release testing, make sure to include the label (ex. 23.0.0-testing) to the PR.
+
+**Reviewers/Assignees:** Reviewers are people that you would like to ping specifically, but are not the same as assignees.
+Be sure to consult the Contributing and Code Review guidelines. For example, 2 reviewers must approve the PR and any code change must be pulled and tested by a developer before a PR is marked as `Passed Manual Tests`. 
+
+**Travis:** The Travis build status and results are linked near the bottom of your PR in GitHub. 
+You can restart Travis if it gives error messages that seem inappropriate, or if the build stalls (for longer than 1h).
+if you are pushing a last typo fix on a PR, it is okay to add `[skip ci]` in the commit message. It is not okay to do this if it's the first commit or if you are fixing more than documentation typos.
```

Comment:
I think adding it under the "Good practice for Travis" is a good idea. It would help looking up anything related to Travis easier in case someone is looking for a quick reference.

Reply:
Sounds good! Thanks